src/misc/misc.c

```diff
@@ -35,8 +35,9 @@ void error(const char* fmt, ...)
 {
 	va_list ap;
 	va_start(ap, fmt);
-	debug_printf(DP_ERROR, "Error: ");
-	debug_vprintf(DP_ERROR, fmt, ap);
+	if (false == debug_logging)
+		debug_printf(DP_ERROR, "Error: ");
+	debug_printf(DP_ERROR, fmt, ap);
```

Comment:
Changed to debug_vprintf and to !debug_logging. I prefer to use the exclamation point notation, but I was following previous convention (see e.g. src/pics.c)